core/src/test/java/nl/nn/adapterframework/pipes/ExceptionPipeTest.java

```diff
@@ -0,0 +1,54 @@
+package nl.nn.adapterframework.pipes;
+
+import nl.nn.adapterframework.core.PipeRunException;
+import org.junit.Test;
+import org.junit.Before; 
+import org.junit.After;
+
+import java.nio.channels.Pipe;
```

Comment:
> Try not to add any unused imports.
> You can have a look at Codacy (after pushing your changes, it will be available on your PR).

Yes! I looked at it and it indeed gives nice comments on the code quality. I'm going to adjust the code accordingly